site/source/docs/compiling/WebAssembly.rst

```diff
@@ -37,6 +37,7 @@ To use fastcomp, just use the emsdk normally to get ``latest``. For the upstream
 There are some differences you may notice between the two backends, if you upgrade from fastcomp to upstream:
 
 * The wasm backend is strict about linking files with different features sets - for example, if one file was built with atomics but another was not, it will error at link time. This prevents possible bugs, but may mean you need to make some build system fixes.
+* ``WASM=0`` behaves differently in the two backends. In fastcomp we emit asm.js, while in upstream we emit JS (as we want to support all wasm, not all of which can work in asm.js). Also, the implementation of the JS support is to make it look like wasm as much as possible to the outside APIs, so in particular startup will be async just like wasm by default, and you can control that with ``WASM_ASYNC_COMPILATION`` (even though ``WASM=0``).
```

Comment:
I think the parenthetical would be clearer as "(not all wasm constructs can be expressed in asm.js)" or maybe something even more specific than that.

Reply:
Also instead of "the implementation... is to make it look like wasm", maybe "aims to behave the same" or "implement the same external API" or something